requirements.build.txt

```diff
@@ -0,0 +1,12 @@
+fbs @ git+https://github.com/X-Sam/fbs@3586f1d
```

Comment:
Before merging this PR, we have to change this to the tag/commit which results out of this PR: https://github.com/AllYarnsAreBeautiful/fbs/pull/6